examples/ceph-conf.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# This is a sample configmap that helps define a Ceph configuration as required
+# by the CSI plugins.
+
+# Sample ceph.conf available at
+# https://github.com/ceph/ceph/blob/master/src/sample.ceph.conf Detailed
+# documentation is available at
+# https://docs.ceph.com/en/latest/rados/configuration/ceph-conf/
+apiVersion: v1
+kind: ConfigMap
+data:
+  ceph.conf: |-
+    [global]
+    auth_cluster_required = cephx
+    auth_service_required = cephx
+    auth_client_required = cephx
+
+    # Workaround for http://tracker.ceph.com/issues/23446
+    fuse_set_user_groups = false
+
+    # ceph-fuse which uses libfuse2 by default has write buffer size of 2KiB
+    # adding 'fuse_big_writes = true' option by default to override this limit
+    # see https://github.com/ceph/ceph-csi/issues/1928
+    fuse_big_writes = true
+  keyring: |-
```

Comment:
Maybe not include a `keyring`?  It might be sufficient to only map the `/etc/ceph/ceph.conf` file from the ConfigMap, and be able to create additional files through the provisioner/node-plugin?

Consider using https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-specific-path-in-the-volume for that.

Reply:
I tried this one when the configmap is mounted the directory becomes readonly and cannot create a keyring file for that reason I added keyring as an entry to the configmap. let me try once again